package.json

```diff
@@ -55,6 +55,7 @@
     "react-dom": "^15.4.2"
   },
   "peerDependencies": {
+    "prop-types": "^15.0.0-0",
     "react": ">=0.12.0 || ^15.0.0-0"
```

Comment:
can you update this peer dep to `"react": "^0.14.0 || ^15.0.0-0 || ^16.0.0-0"` since v12 and v13 dont support `extend React.Component`?